src/components/icons/clock.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+
+const ClockIcon: React.FunctionComponent<{className?: string}> = ({
```

Comment:
this seems like a better patter for using Heroicons in general to me instead of importing or inlining raw SVG.